### DISCUSSION TITLE
Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by abrombo** [pygae/galgebra#500] [Q&A]
I have install galgebra on Ubuntu 22.04 using github -

```
git clone https://github.com/pygae/galgebra.git
cd galgebra
pip install -e .
```

When I try to run terminal examples from the geany editor I get -

```
Traceback (most recent call last):
  File "/home/brombo/galgebra/examples/Terminal/terminal_check.py", line 4, in <module>
    from galgebra.printer import Eprint
ModuleNotFoundError: No module named 'galgebra'
```

When I run a ipynb file (in Google-Chrome) all the packages are found and the code runs and when I run from the terminal command line, "python3 some_galgebra_code.py" the galgebra modules are found.  Note that when I run python code from the geany editor with sympy imports it works.  

Any idea what is going on here.  It started when I upgraded to Ubuntu 22.04.

**Comment by eric-wieser** [comment]
My guess would be that geany is using an old version of python on your system; if you run a program that prints `sys.executable`, you can compare which versions are being used in the different environments.
